docs/cloud/concepts/flows.md

```diff
@@ -39,22 +39,80 @@ mutation($flow: JSON!) {
 
 ## Flow Versions and Archiving <Badge text="GQL"/>
 
-You can control how Cloud versions your Flows by providing a `versionGroupId` whenever you register a Flow (exposed via the `version_group_id` keyword argument in `flow.register`).  Flows which provide the same `versionGroupId` will be considered versions of each other. By default, Flows with the same name in the same Project will be given the same `versionGroupId` and are considered "versions" of each other.  Anytime you register a new version of a flow, Prefect Cloud will automatically "archive" the old version in place of the newly registered flow.  Archiving means that the old version's schedule is set to "Paused" and no new flow runs can be created.
+You can control how Cloud versions your Flows by providing a `versionGroupId` whenever you register a Flow (exposed via the `version_group_id` keyword argument in `flow.register`). Flows which provide the same `versionGroupId` will be considered versions of each other. By default, Flows with the same name in the same Project will be given the same `versionGroupId` and are considered "versions" of each other. Anytime you register a new version of a Flow, Prefect Cloud will automatically "archive" the old version in place of the newly registered Flow. Archiving means that the old version's schedule is set to "Paused" and no new Flow runs can be created.
 
-You can always revisit old versions and unarchive them, if for example you want the same Flow to run on two distinct schedules.  To archive or unarchive a flow, use the following GraphQL mutations:
+You can always revisit old versions and unarchive them, if for example you want the same Flow to run on two distinct schedules. To archive or unarchive a Flow, use the following GraphQL mutations:
 
 ```graphql
 mutation {
-  archiveFlow( input: { flowId: "your-flow-id-here" }) {
+  archiveFlow(input: { flowId: "your-flow-id-here" }) {
     id
   }
 }
 ```
 
 ```graphql
 mutation {
-  unarchiveFlow( input: { flowId: "your-flow-id-here" }) {
+  unarchiveFlow(input: { flowId: "your-flow-id-here" }) {
     id
   }
 }
 ```
+
+## Flow Settings <Badge text="GQL"/>
+
+Prefect Cloud has several insurance policies to ensure Flows run healthily and robustly. Three such policies are:
+
+- Flow and task run heartbeats
+- [Lazarus resurrections](lazarus-process.html)
+- and version locking
+
+These safeguards can be disabled on a Flow-by-Flow basis using Flow settings.
+
+::: warning Disabling safeguards
+Disabling these safeguards can alter fundamental assumptions about how Flows run in Cloud. Be sure to read the docs and understand how each of these settings alters Flow behavior in Cloud.
+:::
+
+### Disable Heartbeats
```

Comment:
Actually, could you call out that this mechanism will only work on Flows registered with Prefect Core 0.8.1+

Reply:
Good shout-- change incoming.

Reply:
Updated.